src/icons/SmallInfoSeal.tsx

```diff
@@ -0,0 +1,53 @@
+export default function () {
+  return (
+    <svg
+      width="33"
+      height="28"
+      viewBox="0 0 33 28"
+      fill="none"
+      xmlns="http://www.w3.org/2000/svg"
+    >
+      <path
+        d="M3.7059 20.9954L13.0842 5.60536C14.643 3.04733 18.357 3.04733 19.9158 5.60536L29.2941 20.9954C30.9184 23.661 28.9998 27.0769 25.8783 27.0769H7.12167C4.00021 27.0769 2.08159 23.661 3.7059 20.9954Z"
+        fill="#1A0259"
+        stroke="#CF6AFF"
+        stroke-linecap="round"
```

Comment:
Remove all `stroke-linecap="round"`, they're set by default, this is a dead code

Reply:
removed, ty